{"timestamp":"2025-08-21T07:29:39.394922","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-08-21T07:29:39.395968","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/fetchin_cleaning.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-08-21T07:29:39.605645Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:39.606370Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:39.607462Z","level":"info","event":"Current task name:stage_weather_dim","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:39.607968Z","level":"info","event":"Dag name:weather_to_snowflake_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:39.606722","level":"info","event":"Executing: PUT file:///home/your_user/growing_csv/weather_dim.csv @%WEATHER_DIM OVERWRITE = TRUE;","logger":"airflow.task.operators.airflow.providers.common.sql.operators.sql.SQLExecuteQueryOperator"}
{"timestamp":"2025-08-21T07:29:39.610975","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py:471: DeprecationWarning: Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `from airflow.sdk import Connection` instead\n  warnings.warn(\n","logger":"py.warnings"}
{"timestamp":"2025-08-21T07:29:39.628610","level":"info","event":"Connection Retrieved 'weather_pollution'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-21T07:29:40.522048","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py:471: DeprecationWarning: Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `from airflow.sdk import Connection` instead\n  warnings.warn(\n","logger":"py.warnings"}
{"timestamp":"2025-08-21T07:29:40.534396","level":"info","event":"Connection Retrieved 'weather_pollution'","logger":"airflow.hooks.base"}
{"timestamp":"2025-08-21T07:29:40.535033","level":"info","event":"Snowflake Connector for Python Version: 3.16.0, Python Version: 3.12.11, Platform: Linux-6.6.87.2-microsoft-standard-WSL2-x86_64-with-glibc2.36","logger":"snowflake.connector.connection"}
{"timestamp":"2025-08-21T07:29:40.535780","level":"info","event":"Connecting to GLOBAL Snowflake domain","logger":"snowflake.connector.connection"}
{"timestamp":"2025-08-21T07:29:41.938102","level":"info","event":"Running statement: PUT file:///home/your_user/growing_csv/weather_dim.csv @%WEATHER_DIM OVERWRITE = TRUE;, parameters: None","logger":"airflow.task.hooks.airflow.providers.snowflake.hooks.snowflake.SnowflakeHook"}
{"timestamp":"2025-08-21T07:29:41.938251","level":"info","event":"Running statement: PUT file:///home/your_user/growing_csv/weather_dim.csv @%WEATHER_DIM OVERWRITE = TRUE;, parameters: None","logger":"airflow.task.hooks.airflow.providers.snowflake.hooks.snowflake.SnowflakeHook"}
{"timestamp":"2025-08-21T07:29:42.635834","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"ProgrammingError","exc_value":"253006: 253006: File doesn't exist: ['/home/your_user/growing_csv/weather_dim.csv']","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":890,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1177,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/operators/sql.py","lineno":312,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/snowflake/hooks/snowflake.py","lineno":547,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":815,"name":"_run_command"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/cursor.py","lineno":1090,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/file_transfer_agent.py","lineno":393,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/file_transfer_agent.py","lineno":999,"name":"_init_file_metadata"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":279,"name":"errorhandler_wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":334,"name":"hand_to_other_handler"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/snowflake/connector/errors.py","lineno":210,"name":"default_errorhandler"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-08-21T07:29:42.638481Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:42.638885Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:42.639239Z","level":"info","event":"Task:<Task(SQLExecuteQueryOperator): stage_weather_dim>","chan":"stdout","logger":"task"}
{"timestamp":"2025-08-21T07:29:42.639501Z","level":"info","event":"Failure caused by 253006: 253006: File doesn't exist: ['/home/your_user/growing_csv/weather_dim.csv']","chan":"stdout","logger":"task"}
